docs/src/data/links.tsx

```diff
@@ -619,7 +619,7 @@ export const gettingStarted: ComponentNavItem[] = [
   {
     href: '/getting-started/migration',
     label: 'Migration',
-    platforms: ['react', 'vue', 'angular'],
+    platforms: ['react', 'vue', 'angular', 'react-native'],
```

Comment:
oops, were we hiding the migration page for react native?

Reply:
yeah :(